src/docs/product/integrations/source-code-mgmt/github/index.mdx

```diff
@@ -248,9 +254,16 @@ Stack trace linking takes you from a file in your Sentry stack trace to that sam
 
         - ex. For example, everything after the branch (`main`) matches the file path of `code.py` using a source code path of `https://github.com/MeredithAnya/testing/blob/main/code.py` so you don't need to set a **stack trace root** and **source code root**.
 
-     1. If the filename in your Sentry stack trace frame doesn't match the path to your source code, you will need to replace the **stack_root** part of the filename with your **source_root** to make the filename match the source code path.
+     2. If the filename in your Sentry stack trace frame doesn't match the path to your source code, you will need to replace the **stack_root** part of the filename with your **source_root** to make the filename match the source code path.
         - ex. For example, to get `src/code.py` to match `code.py` when the source code path is `https://github.com/MeredithAnya/testing/blob/main/code.py`, change the **stack trace root** to be set as `src/`, and leave **source code root** empty.
 
+#### Stacktrace modal
```

Comment:
Since this is information about an interaction you're going to have on the Issue Details page, this should probably be in that content, rather than here. But let's discuss offline. I'm not clear on this strategy.